runtime/src/proposals_configuration/defaults.rs

```diff
@@ -357,3 +357,17 @@ pub(crate) fn freeze_pallet_proposal() -> ProposalParameters<BlockNumber, Balanc
         constitutionality: 1,
     }
 }
+
+// Proposal parameters for the 'Decrease Council' proposal
+pub(crate) fn decrease_council_budget() -> ProposalParameters<BlockNumber, Balance> {
+    ProposalParameters {
+        voting_period: days!(3),
+        grace_period: 0,
+        approval_quorum_percentage: TWO_OUT_OF_THREE,
+        approval_threshold_percentage: TWO_OUT_OF_THREE,
+        slashing_quorum_percentage: ALL,
+        slashing_threshold_percentage: ALL,
+        required_stake: Some(dollars!(50)),
+        constitutionality: 1,
```

Comment:
We need to adjust those, will get back to you

Reply:
Council has approved a signal to make this 3/3 approval, 1 constitutionality: https://pioneer.joyutils.org/#/proposals/preview/797

Reply:
Let me know if also the quorum percentage is correct